package.json

```diff
@@ -1,16 +1,23 @@
 {
   "name": "npmgraph",
   "version": "2.7.1",
+  "type": "module",
   "author": {
     "name": "Robert Kieffer",
     "url": "http://github.com/broofa",
     "email": "robert@broofa.com"
   },
   "license": "MIT",
+  "browserslist": [
+    "since 2017-06"
```

Comment:
Why so specific?

Reply:
It's the recommended solution to [this error](https://flaviocopes.com/parcel-regeneratorruntime-not-defined/), which I was running into.